kebechet/managers/initialization/initialization.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+# Kebechet
+# Copyright(C) 2019 Ronan Souza
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+import os
+import typing
+
+from kebechet.managers.manager import ManagerBase
+from kebechet.utils import cloned_repo
+from .util import _GIT_TOKEN_VARIABLE, _GIT_BRANCH_NAME, _GIT_COMMIT_MESSAGE, _GIT_MR_BODY, _GIT_MR_LABELS, \
+    _KEBECHET_CONFIG_FILE_NAME, create_config_file
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class InitManager(ManagerBase):
+    """Manager for initializing Kebechet configs"""
+
+    def run(self, repo_path: str = None, token: str = None, service_type: str = None, slug: str = None)\
```

Comment:
The backlash should not be here. Again, recommend using the [black](https://github.com/psf/black) tool.